.appveyor.yml

```diff
@@ -27,6 +27,12 @@ environment:
     - PYTHON: C:\Python37-x64
       NP_BUILD_DEP: "numpy==1.14.5"
```

Comment:
Numpy build time dependency got bumped to 0.14 for all builds

https://github.com/scikit-image/scikit-image/blob/master/requirements/build.txt#L3

Reply:
So this is correct, then, right?

Reply:
The ones above it are set to 0.13 i think.

Reply:
Gotcha, fixed.